cross/berkeleydb/Makefile

```diff
@@ -1,12 +1,14 @@
 PKG_NAME = db
-PKG_VERS = 6.2.23
```

Comment:
Please do not downgrade package because of impact on other packages.
I propose you create `cross/berkeleydb-5.3` with same content and make `python` depends on this specific version.

Reply:
@Diaoul What do you think? I checked and these other 2 packages need v2 or v3 of `berkeleydb`, so they won't be affected.
We could also just leave it out, since this has been broken since 2012 and nobody really noticed. 
Creating an extra cross-package to the already giant list seems a bit much, or maybe that's just me?

Reply:
(It hasn't been noticed because if it's not available the `dbm` python module still works but is slower: https://docs.python.org/2.7/library/anydbm.html#module-anydbm)